### PR TITLE
feat: add config schema for rollbar plugins

### PR DESCRIPTION
**Pull request by andrewthauer**
## Hey, I just made a Pull Request!

Fixes part of https://github.com/backstage/backstage/issues/3364. Also removes rollbar from the `create-app` as it's not a default plugin + updated readme's

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by benjdlambert** [comment]
@Rugvip does this look good to you - not sure what I'm looking for in the this config schema stuff. Looks good to me though

File: plugins/rollbar-backend/README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ The following values are read from the configuration file.
 
 ```yaml
 rollbar:
-  organization: organization-name
```
Comment (freben) [inline plugins/rollbar-backend/README.md:11]: i wonder if it's confusing to have a pair of frontend+backend plugins, where their installation instructions (and required configs) are spread over the two readmes. I mean it kinda is what it is, just thinking out loud

Comment (freben) [reply plugins/rollbar-backend/README.md:11]: But maybe keep this line and add a comment `# used by rollbar frontend plugin`? Or is it expected that the slug variant is more common and it'd be unnecessary to mention here

**Review by freben — APPROVED** [review]
Looks good to me too.

**Comment by freben** [comment]
Meh, this is fine and I'll just merge :)

File: plugins/rollbar-backend/README.md (hunk 1 above)
Comment (andrewthauer) [reply plugins/rollbar-backend/README.md:11]: At some point I'd like to loop back and deprecate the backend in favour of using the proxy. However, there still are some open questions in general about plugin pairs. For instance, how do they share code.

**Pull request merged**